source/manual/update-the-autocomplete-denylist.html.md

```diff
@@ -0,0 +1,28 @@
+---
+owner_slack: "#govuk-developers"
+title: Update the autocomplete denylist
```

Comment:
Suggest changing this to: "Update the search autocomplete denylist"

Reply:
I dropped using search in it as it seemed a bit repetitive in the index page as it was already in the context of "GOV.UK Search" so the word "search" seemed a bit redundant. Can see why it could be better in title though, so will try it out and check it still reads ok on index page.